microprofile/testing/junit5/src/main/java/module-info.java

```diff
@@ -19,14 +19,19 @@
  */
 module io.helidon.microprofile.testing.junit5 {
 
-    requires io.helidon.microprofile.cdi;
     requires io.helidon.config.mp;
     requires io.helidon.config.yaml.mp;
-    requires org.junit.jupiter.api;
+    requires io.helidon.microprofile.server;
```

Comment:
This is not correct.
We cannot (and do not want to) depend on `microprofile-server`. 
this should be (as I wrote before):
```
requires io.helidon.microprofile.cdi;
requires static io.helidon.microprofile.server;
```

Reply:
ok